docs/guides/kubernetes-dashboard.md

```diff
@@ -14,359 +14,100 @@ The following guide covers how to secure [Kubernetes Dashboard] using Pomerium.
-
-## Dashboard
+## Install Kubernetes Dashboard
 
 [Kubernetes Dashboard] is a general purpose, web-based UI for Kubernetes clusters. It allows users to manage applications running in the cluster and troubleshoot them, as well as manage the cluster itself.
 
-![kubernetes dashboard login page](./img/k8s-dashboard-login.png)
-
-### Install
-
-As with the previous steps, we can use [Helm] to install our instance of [Kubernetes Dashboard].
-
-```sh
-helm install \
-  helm-dashboard \
-  stable/kubernetes-dashboard \
-  --set ingress.enabled="false" \
-  --set enableSkipLogin="true"
-```
-
-That's it. We've now configured kubernetes dashboard to use the default service account, if none-is provided. We've also explicitly told helm that we are going to deploy our own custom, nginx / Pomerium / cert-manager enabled ingress.
-
-## Pomerium
-
-Pomerium is an identity-aware access proxy that can used to serve as an identity-aware reverse proxy, or as a forward-auth provider.
-
-### Configure
-
-Before installing, we will configure Pomerium's configuration settings in `values.yaml`. Other than the typical configuration settings covered in the quick-start guides, we will add a few settings that will make working with Kubernetes Dashboard easier.
-
-We can retrieve the token to add to our proxied policy's authorization header as follows.
+1. Use [Helm] to install a new instance of [Kubernetes Dashboard].
 
-```sh
-$ kubectl describe secret helm-dashboard
-```
+  ```bash
+  helm repo add kubernetes-dashboard https://kubernetes.github.io/dashboard/
+  helm install kubernetes-dashboard kubernetes-dashboard/kubernetes-dashboard\
+    --set ingress.enabled="false"
+  ```
 
-```bash
-Name:         dashboard-kubernetes-dashboard-token-bv9jq
-Namespace:    default
-Labels:       <none>
-Annotations:  kubernetes.io/service-account.name: dashboard-kubernetes-dashboard
-              kubernetes.io/service-account.uid: 18ab35ee-eca1-11e9-8c75-025000000001
+That's it. We've now configured kubernetes dashboard in our cluster. We've also explicitly told Helm that we are going to deploy our own custom access to the service through Pomerium, instead of a standard ingress.
 
-Type:  kubernetes.io/service-account-token
+## Add an Route
 
-Data
-====
-ca.crt:     1025 bytes
-namespace:  7 bytes
-token:      eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.......
+Following the configuration defined in [Install Pomerium using Helm], add a route for the Kubernetes Dashboard.
 
-```
+1. Modify `pomerium-values.yaml` with the following route:
 
-The above token then needs to be assigned to our route configuration and policy.
+    ```yaml
+        - from: https://dashboard.localhost.pomerium.io
+          to: https://kubernetes-dashboard.default.svc
```

Comment:
```suggestion
          to: https://kubernetes-dashboard.default.svc.cluster.local
```

This chart could be installed anywhere but the fqdn to default namespace is a good guess